packages/desktop/src/document.rs

```diff
@@ -25,6 +30,46 @@ impl Document for DesktopDocument {
     fn set_title(&self, title: String) {
         self.desktop_ctx.window.set_title(&title);
     }
+
+    /// Create a new meta tag in the head
+    fn create_meta(&self, props: MetaProps) {
+        let myself = self.clone();
+        queue_effect(move || {
```

Comment:
Ideally these would be a default implementation, but it requires a Clone bound on the document trait for queue_effect which is a breaking change